src/main/java/apprenticeship/App.java

```diff
@@ -1,13 +1,24 @@
 package apprenticeship;
 
+import java.util.Arrays;
+
 /**
  * Hello world!
  *
  */
```

Comment:
Get rid of these comments. You should try to avoid comments in the code unless they are absolutely necessary.